index.js

```diff
@@ -8,6 +8,7 @@ var through    = require('through');
 
 function reactify(filename, options) {
   options = options || {};
+  options.extensions = options.hasOwnProperty('extensions') ? options.extensions : [ '.jsx' ];
```

Comment:
Seems like jsx is the suggested extension nowadays - but happy/easy to change that.